network/vpc/reference-content/vpc-migration.mdx

```diff
@@ -152,9 +152,9 @@ Read our [blog post](https://www.scaleway.com/en/blog/virtual-private-cloud-publ
 
 ## Further resources
 
-- [VPC Public Beta Documentation](https://www.scaleway.com/en/docs/network/vpc/)
-- [VPC FAQ](https://www.scaleway.com/en/docs/faq/vpc/)
+- [VPC Public Beta Documentation](https://www.scaleway.com/docs/network/vpc/)
+- [VPC FAQ](https://www.scaleway.com/docs/faq/vpc/)
 - [VPC API Documentation](https://www.scaleway.com/en/developers/api/vpc/)
 - [Scaleway Developers Tools](https://www.scaleway.com/en/developers/)
-- [Tutorials](https://www.scaleway.com/en/docs/tutorials/?facetFilters=%5B%22categories%3Avpc%22%5D&page=1)
+- [Tutorials](https://www.scaleway.com/docs/tutorials/?facetFilters=%5B%22categories%3Avpc%22%5D&page=1)
```

Comment:
Shouldn't we remove the` https://www.scaleway.com/docs` part of URL to doc website ?